src/test/java/seedu/address/logic/command/add/AddAccommodationCommandTest.java

```diff
@@ -112,6 +113,13 @@ public void addTravelPlanObject(TravelPlanObject travelPlanObject) {
             accommodationsAdded.add(accommodation);
         }
 
+        @Override
```

Comment:
Not this line, but for line 97, can change "person" in the javadoc comment to accommodation?